.github/workflows/on-push-master.yml

```diff
@@ -0,0 +1,56 @@
+name: Run coveralls and publish to gh-pages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v2
+
+    - name: Use Node.js 14.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+
+    - name: Cache Node.js modules 💾
+      uses: actions/cache@v2
```

Comment:
Can we please also cache the contents of `resources/external`? I understand that `actions/cache@v2` caches the local `node_modules` directory?

Reply:
Done in 5e6e76c1